backend/data_distro/api_tests/schema/validate_log.txt

```diff
@@ -0,0 +1,1456 @@
+{'verbosity': 1, 'settings': None, 'pythonpath': None, 'traceback': True, 'no_color': False, 'force_color': False, 'skip_checks': False, 'view': 'vw_findings', 'year': 2022, 'start': 0, 'end': 100, 'step': None}
```

Comment:
@eazapata23 I left this file in here to give you an idea about what you should see coming back from the validator. The first line should always give you a readout of the arguments you're passing to the validator.